packages/core/src/utils/create/index.js

```
@@ -270,128 +270,134 @@
+            response: HttpResponse, requestInfo, status,
+            ...HttpResponse
+          }
+          window.postRequest && window.postRequest(event);
+          let body =  event?.response?.body || event?.body
+          try {
+            response.data  =  JSON.parse(body)
+          } catch (error) {
+            response.data = body
+          }
+          response.headers = event?.response?.headers || event?.headers
+          return response;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)